app/prices.js

```diff
@@ -56,28 +56,30 @@ exports.init = function (callback) {
                 continue;
             }
 
+            const attributes = getAttributes(pricelist[i].sku);
+
             // Go through pricestf prices
-            for (let j = 0; j < prices.length; j++) {
-                if (pricelist[i].name === prices[j].name) {
+            for (let j = 0; j < prices[attributes.quality][attributes.killstreak].length; j++) {
+                const item = prices[attributes.quality][attributes.killstreak][j];
+
+                if (pricelist[i].name === item.name) {
                     // Found matching items
-                    if (prices[j].buy !== null && pricelist[i].time < prices[j].time) {
+                    if (pricelist[i].time < item.time) {
                         // Times don't match, update our price
-                        pricelist[i].buy = new Currencies(prices[j].buy);
-                        pricelist[i].sell = new Currencies(prices[j].sell);
-                        pricelist[i].time = prices[j].time;
+                        pricelist[i].buy = new Currencies(item.buy);
+                        pricelist[i].sell = new Currencies(item.sell);
+                        pricelist[i].time = item.time;
```

Comment:
Since you use the item variable for the pricestf pricelist, then you could have made a variable for our own pricelist too, but it is fine.